### PR TITLE
Kirundi translation corrections

### PR DESCRIPTION
**Pull request by Diama1**
#923

### CODE REVIEW

**Comment by anvmn** [comment]
@Diama1 what's the status here?

**Comment by Diama1** [comment]
> @Diama1 what's the status here?

@anvmn I waited for the Burundi team to give more translations but they didn't and i reached out and seems like that was what they could give so far. So on my end it is done

**Comment by anvmn** [comment]
@Diama1 

Ok then. I see there are some conflicts.
Please resolve and then assign me for a review.

**Comment by anvmn** [comment]
@Diama1 ^^

File: client/src/elm/Gizra/NominalDate.elm [hunk 1]
```diff
@@ -95,8 +95,9 @@ fromLocalDateTime =
 
 {-| Given a date, return date representing it's last month day.
 
-    toLastDayOfMonth 2019-08-01" --> 2019-08-31"
-    toLastDayOfMonth 2017-02-20" --> 2017-02-28"
+    toLastDayOfMonth 2019 - 8 - 1 " --> 2019-08-31"
```
Comment (anvmn) [inline client/src/elm/Gizra/NominalDate.elm]: Revert this

File: client/src/elm/Translate.elm [hunk 2]
```diff
@@ -12166,13 +12166,13 @@ translationSet trans =
...
                     { english = "Head Circumference"
                     , kinyarwanda = Just "Umuzenguruko w'umutwe"
-                    , kirundi = Just "Umuzingi w'umutwe"
+                    , kirundi = Just "Intikuro y'umutwe"
                     }
 
                 TaskMuac ->
                     { english = "MUAC"
                     , kinyarwanda = Just "Ikizigira cy'akaboko"
-                    , kirundi = Just "(CMB) Uburyo bwo gupima vuba na vuba ingwara yo gufungura nabi"
```
Comment (anvmn) [inline client/src/elm/Translate.elm:12175]: "MUAC" appears at many places. Fix main one, and use `translationSet` for all others.

File: client/src/elm/Translate.elm [hunk 3]
```diff
@@ -13264,7 +13264,10 @@ translationSet trans =
         PrenatalActivityTitle activity ->
             case activity of
                 Backend.PrenatalActivity.Model.DangerSigns ->
-                    translationSet DangerSigns
```
Comment (anvmn) [inline client/src/elm/Translate.elm:13267]: revert this

**Review by anvmn — CHANGES_REQUESTED** [review]

File: client/src/elm/Translate.elm (hunk 3 above)
Comment (Diama1) [reply client/src/elm/Translate.elm:13267]: Here on prenatal the Kirundi translation is different from the other one.

File: client/src/elm/Gizra/NominalDate.elm (hunk 1 above)
Comment (Diama1) [reply client/src/elm/Gizra/NominalDate.elm]: removing the spacing is failing lint test

Comment (anvmn) [reply client/src/elm/Gizra/NominalDate.elm]: @Diama1 

1. Revert to original state, and lets see if it still fails.
2. Error says file is not formatted properly with `elm-format` 0.8.7. So make sure you have `elm-format` 0.8.7 installed locally.

File: client/src/elm/Translate.elm (hunk 3 above)
Comment (anvmn) [reply client/src/elm/Translate.elm:13267]: Ok, add a comment about it next to this translation.

File: client/src/elm/Translate.elm [hunk 4]
```diff
@@ -13264,7 +13264,10 @@ translationSet trans =
...
+                    { english = "Danger Signs"
+                    , kinyarwanda = Just "Ibimenyetso mpuruza"
+                    , kirundi = Just "Imbanyi igeramiwe"
+                    }
```
Comment (Diama1) [inline client/src/elm/Translate.elm:13256]: The Kirundi translation for prenatal danger signs is different from the  general translations for danger signs.

**Comment by Diama1** [comment]
@anvmn same error and when I run `elm-format ./src/elm --yes` it puts the spacing.

File: client/src/elm/Gizra/NominalDate.elm (hunk 1 above)
Comment (Diama1) [reply client/src/elm/Gizra/NominalDate.elm]: @anvmn same error and when I run `elm-format ./src/elm --yes` it puts the spacing.

Comment (anvmn) [reply client/src/elm/Gizra/NominalDate.elm]: @Diama1 

You have not reverted Nominal date change. We should not be seeing this in PR:

Selection_327

Comment (Diama1) [reply client/src/elm/Gizra/NominalDate.elm]: done

**Review by anvmn — COMMENTED** [review]
@Diama1 

Did you change translation of main MUAC as needed?

**Comment by Diama1** [comment]
> @Diama1 
> 
> 
> 
> Did you change translation of main MUAC as needed?

Yeah on the first commit named fix

File: client/src/elm/Translate.elm (hunk 2 above)
Comment (Diama1) [reply client/src/elm/Translate.elm:12175]: fixed

**Comment by anvmn** [comment]
> > @Diama1
> > Did you change translation of main MUAC as needed?
> 
> Yeah on the first commit named fix

This is what I see now. Is this what it should be?

Selection_330

**Review by anvmn — APPROVED** [review]

**Pull request merged**